searchlib/src/tests/common/bitvector/bitvector_test.cpp

```diff
@@ -844,4 +844,56 @@ TEST(BitvectorTest, fixup_count_and_guard_bit_and_zero_remaining_data_bits_after
     EXPECT_TRUE(bv == bv2);
 }
 
+TEST(BitvectorTest, normal_guard_bits)
+{
+    for (uint32_t i = 0; i < 2; ++i) {
```

Comment:
Would it make sense to have this loop be bounded by a number derived from `BitVector::num_guard_bits` instead? Also applies to the test in the other loop.